ArjunaJTS/orbportability/classes/com/arjuna/orbportability/common/ant/IDLCompiler.java

```diff
@@ -84,12 +84,12 @@ public IDLCompiler()
 
     public final void setDebug(String debug)
     {
-        _debugOn = new Boolean(debug).booleanValue();
+        _debugOn = Boolean.valueOf(debug).booleanValue();
```

Comment:
You don't need to unbox the boolean value, ie `_debugOn = Boolean.valueOf(debug)` is sufficient.
Similarly elsewhere in this PR.

Also `Boolean.valueOf(debug)` assigns a boxed value to the primitive `_debugOn` so incurs unboxing.

So to summarise the best approach is to just do `_debugOn = Boolean.parseBoolean(debug);` 

Similarly elsewhere in this PR (ie I won't repeat the same comment).